pages/about.html

```diff
@@ -2,11 +2,43 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
+  <meta type="author" content="Rae Elwell">
+  <meta type="description" content="Rae's Corner">
+  <title>About Me</title>
+  <link rel="stylesheet" href="/style.css">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
 </head>
 <body>
-  
+  <nav class="nav-block">
+    <div class="intro">
+      <h1>About Me</h1>
+  </div>
```

Comment:
A `<header>` would have been more appropriate here:

```suggestion
    <header class="intro">
      <h1>About Me</h1>
    </header>
```